src/zkevm_specs/evm/execution_result/call.py

```diff
@@ -0,0 +1,156 @@
+from ...util import linear_combine, le_to_int, EMPTY_CODE_HASH
+from ..common_assert import assert_bool
+from ..step import Step
+from ..table import FixedTableTag, RWTableTag, CallContextTag
+from ..opcode import Opcode
+from .execution_result import ExecutionResult
+
+
+def call(curr: Step, next: Step, r: int, opcode: Opcode):
+    # Verify opcode
+    assert opcode == Opcode.CALL
+
+    # Verify depth
+    depth = curr.call_context_lookup(CallContextTag.Depth)
+    curr.fixed_lookup(FixedTableTag.Range1024, [depth])
```

Comment:
I think valid depth maximum can be  1024 according to https://github.com/ethereum/go-ethereum/blob/master/core/vm/evm.go#L172  , the look up range should be Range1025 , pls. check it !Thx

Reply:
Thanks for catching this! Depth starts from `1` and would always be propagated by caller, so let's check it's in range by `curr.fixed_lookup(FixedTableTag.Range1024, [depth - 1])`.